src/View.elm

```diff
@@ -0,0 +1,178 @@
+module View exposing (viewStyled)
+
+import Css exposing (..)
```

Comment:
What are your thoughts on elm-css vs pure CSS/SCSS vs elm-ui vs Tailwind vs perhaps some other options? wrt. maintainability, safety, ease of use etc...

Reply:
elm-css: it's css with compiler/IDE assist, a no brainer. Pure css comes close. I think because of the visual nature of every css line in code, it's hard to make big mistakes with it.
Elm-ui: This should have never become a meme. I think there's a lot of knee jerk regarding CSS because it's such a huge language, which is warranted, but web developers really should know CSS well anyway, at which point the only selling point of elm-ui - which is that it's not CSS - vanishes. I've seen a commercial project getting rewritten from scratch because elm-ui proved to be more trouble than utility. In my opinion, CSS is vastly underappreciated by the elm-ui crowd.
Tailwind is fine, saves a few characters at the cost of deviating from the lingua franca. I haven't used SCSS much, and I think elm-css is a direct competitor of it, and it is a much better fit with the Elm ecosystem.